mmtrack/models/trackers/tao_tracker.py

```diff
@@ -0,0 +1,368 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+from collections import defaultdict
+
+import torch
+from mmdet.core import bbox_overlaps
+
+from mmtrack.core import embed_similarity, imshow_tracks
+from ..builder import TRACKERS
+from .base_tracker import BaseTracker
+
+
+@TRACKERS.register_module()
+class TaoTracker(BaseTracker):
```

Comment:
Rename it to `QuasiDenseTAOTracker`.
And rename `QuasiDenseEmbedTracker` to `QuasiDenseMOTTracker`

Reply:
ok, but the QuasiDenseEmbedTracker is used by datasets other than TAO, like bdd and mot17, should we change the name to mot?

Reply:
Then, rename `TaoTracker` to `QuasiDenseTAOTracker`, and `QuasiDenseEmbedTracker` to ``QuasiDenseTracker

Reply:
And remember to add more comments in the docstring of `QuasiDenseTAOTracker` and `QuasiDenseTracker` to explain that the `QuasiDenseTAOTracker` is for TAO and `QuasiDenseTracker` is for MOT Challenge and BDD100K.